src/olympia/ratings/tests/test_models.py

```diff
@@ -240,7 +240,7 @@ def test_reply_triggers_email_but_no_logging(self):
             body='Rêply',
         )
 
-        assert not ActivityLog.objects.exists()
+        assert ActivityLog.objects.exists()
```

Comment:
lets check the object is correct here too (see ` test_edit_triggers_logging_but_no_email` below for roughly what to check)